packages/react-start-rsc/src/plugin/vite.ts

```
@@ -1,49 +1,15 @@
 import { fileURLToPath } from 'node:url'
 import path from 'pathe'
-import { resolveViteId } from '@tanstack/start-plugin-core/utils'
-import type {
-  TanStackStartVitePluginCoreOptions,
-  ViteRscForwardSsrResolverStrategy,
-} from '@tanstack/start-plugin-core/vite/types'
-import type { Plugin, PluginOption, UserConfig } from 'vite'
+import {
+  createVirtualModule,
+  type TanStackStartVitePluginCoreOptions,
```

ESLint

[error] Member 'TanStackStartVitePluginCoreOptions' of the import declaration should be sorted alphabetically.

(sort-imports)